api/predict.py

```diff
@@ -73,9 +73,17 @@ def post(self):
         result = {'status': 'error'}
 
         input_json = MAX_API.payload
-        for p in input_json["paragraphs"]:
-            if p["context"] == "":
-                abort(400, "Empty input, please provide a paragraph.")
+        try:
+            for p in input_json["paragraphs"]:
+                assert frozenset(p.keys()) == frozenset(["context", "questions"])
```

Comment:
1) Is there a case where the assert would be triggered?
2) If there is, an assert is not going to yield the expected result (HTTP code 400)

Reply:
It is when you pass an invalid key like `question` instead of `questions`. I have tested it out. 
![image](https://user-images.githubusercontent.com/32885872/65173984-daf93400-da04-11e9-9212-14bc079583d3.png)